src/solve/client/__tests__/client.test.ts

```diff
@@ -181,4 +190,127 @@ describe("SkipClient", () => {
       ).resolves.toEqual(expect.anything());
     });
   });
+
+  describe("executeMultiChainMessage", () => {
+    it("can execute transfer message", async () => {
+      const wallet = await DirectSecp256k1HdWallet.fromMnemonic(
+        "enlist hip relief stomach skate base shallow young switch frequent cry park"
+      );
+
+      const accounts = await wallet.getAccounts();
+      const address = accounts[0].address;
+
+      const message = {
+        chain_id: "cosmoshub-localnet-1",
+        path: ["cosmoshub-4", "osmosis-1"],
+        msg: `{"source_port":"transfer","source_channel":"channel-141","token":{"denom":"uatom","amount":"5000"},"sender":"${address}","receiver":"osmo1f2f9vryyu53gr8vhsksn66kugnxaa7k8jdpk0e","timeout_height":{},"timeout_timestamp":1690975247072565510}`,
+        msg_type_url: "/ibc.applications.transfer.v1.MsgTransfer",
+      };
+
+      const client = new SkipClient({
+        endpointOptions: {
+          "cosmoshub-localnet-1": {
+            rpc: "localhost:26657",
+          },
+        },
+      });
+
+      const tx = await client.executeMultiChainMessage(
+        address,
+        wallet,
+        message,
+        coin(0, "uatom")
+      );
+
+      // // CheckTx must pass but the execution must fail in DeliverTx due to invalid channel/port
+      expect(isDeliverTxFailure(tx)).toEqual(true);
```

Comment:
I know this seems weird. 

See https://github.com/cosmos/cosmjs/blob/main/packages/stargate/src/signingstargateclient.spec.ts#L269

It would be cool to get a full IBC network up in the future though